src/Sylius/Bundle/CoreBundle/Sitemap/Exception/SitemapUrlNotFoundException.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace Sylius\Bundle\CoreBundle\Sitemap\Exception;
+
+use Sylius\Bundle\CoreBundle\Sitemap\Model\SitemapUrlInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class SitemapUrlNotFoundException extends \Exception
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(SitemapUrlInterface $sitemapUrl)
```

Comment:
It would be nice to add ability to chain exceptions. `__construct(SitemapUrlInterface $sitemapUrl, \Exception $previousException = null)`